<?xml version="1.0"?> 

<launch>

    <!-- Start costmap node for obstacles -->
    <node pkg="costmap_2d" type="costmap_2d_node" name="obstacle_costmap" output="screen">
        <rosparam command="load" file="$(find vector_field_histogram)/param/obstacle_costmap_params.yaml" ns="/obstacle_costmap/costmap"/>
    </node>

    <!-- Start costmap node for left lane -->
    <node pkg="costmap_2d" type="costmap_2d_node" name="lane_costmap" output="screen">
        <rosparam command="load" file="$(find vector_field_histogram)/param/lane_costmap_params.yaml" ns="/lane_costmap/costmap"/>
    </node>

    <!-- Start the node to combine the costs (choosing the maximum) -->
    <node pkg="vector_field_histogram" type="composite_costmap.py" name="composite_costmap" output="screen" />

</launch>
